dependency_cache.go

```diff
@@ -71,12 +70,13 @@ type DependencyCache struct {
 
 // NewDependencyCache creates a new instance setting the default cache path (<BUILDMODULE_PATH>/dependencies) and user
 // agent (<BUILDMODULE_ID>/<BUILDMODULE_VERSION>).
-func NewDependencyCache(buildModuleID string, buildModuleVersion string, buildModulePath string, platformBindings libcnb.Bindings) (DependencyCache, error) {
+func NewDependencyCache(buildModuleID string, buildModuleVersion string, buildModulePath string, platformBindings libcnb.Bindings, logger log.Logger) (DependencyCache, error) {
 	cache := DependencyCache{
 		CachePath:    filepath.Join(buildModulePath, "dependencies"),
 		DownloadPath: os.TempDir(),
-		UserAgent:    fmt.Sprintf("%s/%s", buildModuleID, buildModuleVersion),
```

Comment:
why is it gone?

Reply:
Not gone, just moved. See a couple of lines down.

It is not a practice that we've previously followed, but it is something I've heard is often done, so I've started putting the properties in alphabetic order. That's why it moved.

Reply:
ah sorry, not sure why I missed that... thanks!